### PR TITLE
Issue #203 - Move to Github Actions

### PR DESCRIPTION
**Pull request by agarwalkartik**
#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message and code follows [Code of conduct](https://github.com/fastify/fastify/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+name: CI workflow
+on: [push, pull_request]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [6.x, 8.x, 10.x, 11.x, 12.x]
```
Comment (mcollina) [inline .github/workflows/ci.yml]:
```suggestion
        node-version: [6.x, 8.x, 10.x, 11.x, 12.x, 13.x]
```

**Comment by agarwalkartik** [comment]
> Would you mind to add a badge to the readme as well?

sure, will do it.

**Comment by Eomm** [comment]
Do we want to remove Travis and azure at all?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - name: Install Dependencies
+      run: npm install
+    - name: Linter
+      run: npm run test:lint
+    - name: Test
+      run: npm run test:report && npm run test:reporter
```
Comment (Eomm) [inline .github/workflows/ci.yml]: This was used by azure as a report, npm test should be enough for github action

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+      matrix:
+        node-version: [6.x, 8.x, 10.x, 11.x, 12.x, 13.x]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - name: Install Dependencies
+      run: npm install
```
Comment (Eomm) [inline .github/workflows/ci.yml]:
```suggestion
      run: npm install --ignore-scripts
```

I'm spreading this install script 👍

**Pull request merged**